lab-saul/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**
This is a top comment to let you know you need an ``.eslintrc`` in your lab per the lab instructions.